src/common/dto/pagination-query.dto.ts

```diff
@@ -0,0 +1,10 @@
+import { IsOptional, IsPositive } from "class-validator";
+export class PaginationQueryDto {
+  @IsOptional()
+  @IsPositive()
+  limit: number;
+
+  @IsOptional()
+  @IsPositive()
+  offset: number;
+}
```

Comment:
I am not sure whether it is a good idea to add an independent directory "dto" in the common directory when other modules will put their own dto in their own directory. I think maybe it should be put into a folder like "utils".